manifest.yml

```diff
@@ -2,7 +2,7 @@
 applications:
   - name: tta-smarthub-((env))
     buildpacks:
-      - https://github.com/cloudfoundry/nodejs-buildpack.git#v1.7.54
+      - nodejs_buildpack
     env:
```

Comment:
@SarahJaine or @jasalisbury will be able to tell me if I'm imagining things, but I think I remember them updating this recently from 'nodejs_buildpack', something to do with CI failures.

Reply:
`nodejs_buildpack` means "the version of the buildpack that cloud.gov supplies". The version can be found by running `cf buildpacks | grep nodejs` in a terminal. And the versions of node supported by that buildpack can be found by looking the version up here: https://github.com/cloudfoundry/nodejs-buildpack/releases

The pro of using this is that we're more in line with the SSP that our base image comes from cloud.gov as opposed to from cloud foundry without the cloud.gov review.

The downside is that sometimes a new version of the buildpack is released that doesn't have NodeJS version overlap with what the app is using, so deploys will fail until you either pin the buildpack version (such as to 1.7.54, here) or update the NodeJS version in use to something in the official buildpack, as this PR does.

Reply:
gotcha, so moving back to the stock buildpack moves the onus on us to keep node updated to the latest supported version.

Reply:
Yes, and/or switching back to a tagged version if need be. But in the meantime there's the benefit of using the officially blessed version.